.dockerignore

```diff
@@ -0,0 +1,33 @@
+.git
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+
+# Virtual environment
+env
+venv
+/build
+
+# Node
+node_modules
+
+# Logs
+*.log
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Application data
+/data
```

Comment:
Yes. It's in the root of the context (git repo). https://docs.docker.com/engine/reference/builder/#dockerignore-file